src/service/resources-impl.js

```diff
@@ -686,9 +686,8 @@ export class Resources {
 
     if (this.documentReady_ && this.firstPassAfterDocumentReady_) {
       this.firstPassAfterDocumentReady_ = false;
-      this.viewer_.postDocumentReady(this.viewport_.getSize().width,
-        this.win.document.body./*OK*/scrollHeight);
-      this.updateScrollHeight_();
```

Comment:
Correct. We need to dump them all, along with `ViewportBindingVirtual_`. Do you want to collapse these changes into this PR? Or I can do it.